web/yo/app/scripts/controllers/gene.js

```diff
@@ -775,16 +812,20 @@ angular.module('oncokbApp')
                 var reviewObj;
                 switch (type) {
                 case 'GENE_SUMMARY':
-                    data.description = $scope.gene.summary.text;
-                    dataUUID = $scope.gene.summary_uuid.getText();
-                    data.lastEdit = $scope.gene.summary_review.get('updateTime');
-                    reviewObj = $scope.gene.summary_review;
+                    if(mainUtils.needReview($scope.gene.summary_uuid)) {
```

Comment:
We didn't need to check needReview before, because it has already been done when they enter the review mode. But now we support approve section change feature. If a section is newly added, we display it as it is without performing needReview check on them. That is why we need to add the check here